modules/installation-mirror-repository.adoc

```diff
@@ -48,13 +87,51 @@ the pull secret for your mirror registry that you created.
 <6> The release mirror. For a production release, you must specify
 `ocp-release`.
 
+. Build the signature for the content that you plan to mirror and verify that it
+is signed by an official Red Hat key:
+.. Set the digest for the version to install to:
++
+----
+$ DIGEST="$(oc adm release info quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE} | sed -n 's/Pull From: //p')" <1>
```

Comment:
I do not think this command would get a correct DIGEST.
`
$ oc adm release info quay.io/openshift-release-dev/ocp-release:4.2.0-rc.2 | sed -n 's/Pull From: //p'

quay.io/openshift-release-dev/ocp-release@sha256:9a1a38ce4d5fa79c785801eca3679a759bede9f40081b9e1b9f0cca2baa673a
`

The DIGEST should be 'sha256:9a1a38ce4d5fa79c785801eca3679a759bede9f40081b9e1b9f0cca2baa673a2', right?

Reply:
Ah, right.  Could use `s/Pull From: .*@//p` to jump straight to the digest